domain.yml

```diff
@@ -124,6 +142,28 @@ responses:
         *3.* OTHER
         *4.* RATHER NOT SAY
 
+  utter_ask_location:
+    - text: |
+        We need to collect your location.
+
+        Please share your location 📍 using WhatsApp by following these steps:
+        - Tap the attach icon (+ or  📎) on the bottom of WhatsApp.
+        - Select location
+        - This will bring up a map with a pin showing your current location. Tap *Send Your Current Location* if this is your address.
+        - You can also choose from various other nearby locations if you think that the GPS is a bit off.
+
+
+        Or TYPE the name of your Suburb, Township, Town or Village (or nearest)?
+
+  utter_ask_location_confirm:
+    - text: "We use Google Location Services to verify locations. Please confirm that the address below is correct based on the information you have shared:\n
+            ----------------------------\n
+            {location}\n
+            ----------------------------\n
+            Reply:\n
+            *1.* YES\n
+            *2.* NO"
```

Comment:
Will this not generate large whitespace at the beginning of lines, vs using the yaml `|` for declaring multiline strings?